package.json

```diff
@@ -76,59 +76,60 @@
-		"@wordpress/hooks": "3.2.2",
-		"@wordpress/html-entities": "3.2.3",
-		"@wordpress/i18n": "4.2.4",
-		"@wordpress/icons": "6.1.1",
-		"@wordpress/interface": "4.1.18",
-		"@wordpress/is-shallow-equal": "4.2.1",
-		"@wordpress/keyboard-shortcuts": "3.0.7",
-		"@wordpress/keycodes": "3.2.4",
-		"@wordpress/list-reusable-blocks": "3.0.20",
-		"@wordpress/media-utils": "3.0.5",
-		"@wordpress/notices": "3.2.8",
-		"@wordpress/nux": "5.0.20",
-		"@wordpress/plugins": "4.0.7",
-		"@wordpress/primitives": "3.0.4",
-		"@wordpress/priority-queue": "2.2.3",
-		"@wordpress/redux-routine": "4.2.2",
-		"@wordpress/reusable-blocks": "3.0.23",
-		"@wordpress/rich-text": "5.0.8",
-		"@wordpress/server-side-render": "3.0.20",
-		"@wordpress/shortcode": "3.2.3",
-		"@wordpress/token-list": "2.2.2",
-		"@wordpress/url": "3.3.1",
-		"@wordpress/viewport": "4.0.7",
-		"@wordpress/warning": "2.2.2",
-		"@wordpress/widgets": "2.0.23",
-		"@wordpress/wordcount": "3.2.3",
+		"@wordpress/a11y": "3.6.0",
+		"@wordpress/annotations": "2.6.0",
+		"@wordpress/api-fetch": "6.3.0",
+		"@wordpress/autop": "3.6.0",
+		"@wordpress/blob": "3.6.0",
+		"@wordpress/block-directory": "3.4.0",
+		"@wordpress/block-editor": "8.5.0",
+		"@wordpress/block-library": "7.3.0",
+		"@wordpress/block-serialization-default-parser": "4.6.0",
+		"@wordpress/blocks": "11.5.0",
+		"@wordpress/components": "19.8.0",
+		"@wordpress/compose": "5.4.0",
+		"@wordpress/core-data": "4.4.0",
+		"@wordpress/customize-widgets": "3.3.0",
+		"@wordpress/data": "6.6.0",
+		"@wordpress/data-controls": "2.6.0",
+		"@wordpress/date": "4.6.0",
+		"@wordpress/deprecated": "3.6.0",
+		"@wordpress/dom": "3.6.0",
+		"@wordpress/dom-ready": "3.6.0",
+		"@wordpress/edit-post": "6.3.0",
+		"@wordpress/edit-site": "4.3.0",
+		"@wordpress/edit-widgets": "4.3.0",
+		"@wordpress/editor": "12.5.0",
+		"@wordpress/element": "4.4.0",
+		"@wordpress/escape-html": "2.6.0",
+		"@wordpress/format-library": "3.4.0",
+		"@wordpress/hooks": "3.6.0",
+		"@wordpress/html-entities": "3.6.0",
+		"@wordpress/i18n": "4.6.0",
+		"@wordpress/icons": "8.2.0",
+		"@wordpress/interface": "4.5.0",
+		"@wordpress/is-shallow-equal": "4.6.0",
+		"@wordpress/keyboard-shortcuts": "3.4.0",
+		"@wordpress/keycodes": "3.6.0",
+		"@wordpress/list-reusable-blocks": "3.4.0",
+		"@wordpress/media-utils": "3.4.0",
+		"@wordpress/notices": "3.6.0",
+		"@wordpress/nux": "5.4.0",
+		"@wordpress/plugins": "4.4.0",
+		"@wordpress/preferences": "1.2.0",
+		"@wordpress/primitives": "3.4.0",
+		"@wordpress/priority-queue": "2.6.0",
+		"@wordpress/redux-routine": "4.6.0",
+		"@wordpress/reusable-blocks": "3.4.0",
+		"@wordpress/rich-text": "5.4.0",
+		"@wordpress/server-side-render": "3.4.0",
+		"@wordpress/shortcode": "3.6.0",
+		"@wordpress/token-list": "2.6.0",
+		"@wordpress/url": "3.7.0",
+		"@wordpress/viewport": "4.4.0",
+		"@wordpress/warning": "2.6.0",
+		"@wordpress/widgets": "2.4.0",
+		"@wordpress/wordcount": "3.6.0",
```

Comment:
We should also updated `react` and `react-dom` to v17.0.2 in this file and here:

https://github.com/WordPress/wordpress-develop/blob/cab6fec683eba7bcecad119d3e9d38850255033b/src/wp-includes/script-loader.php#L106-L107